### ISSUE TITLE
[BUG] phpmyadmin imagen arm64

### ISSUE THREAD

**Issue by xploz1on** [sprintcube/docker-compose-lamp#205]
Hello all.
I suggest changing:
```
phpmyadmin:
    image: phpmyadmin/phpmyadmin
```
Just to:
````
phpmyadmin:
    image: phpmyadmin
````

If someone for example like me is running this on a "arm64" phpmyadmin will fail. Simple changing that makes it work for all architectures.

Other than that, everything is working perfect! My local dev has no issues.
I have a quick question. I will be moving the dev setup to a cloud server. It has a nginx-proxy-manager setup to redirect all subdomains to the containers. Will i need to change something to make it work? 
Thanks!

**Comment by alainseys** [comment]
@xploz1on tested this out on a arm system and phpmyadmin/phpmyadmin has indeed some issues on arm64, the image phpmyadmin is the new repo for phpmyadmin so a change can be executed.

that been said your question about moving the dev setup to a cloud server is not recommended for production usage.
But you can use the nxing proxy manager to handle the subdomain request change the value in the .env file and if everything is setup correctly nginx will handle the rest .

The only thing that could give issues are the ussed ports on nginx proxy these use the default ports (80,443) so change them before applying changes .

**Comment by MrOffline77** [comment]
Hi all,
good point you mentioned here!
Fixed with merge of #208  :)
